src/components/widgets/DeleteFriendModal/DeleteFriendModal.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+import classes from "./DeleteFriendModal.module.css";
+
+interface DeleteFriendModalProps {
+    name?: string;
+    children?: React.ReactNode;
+    companionCounter?: number;
+}
+
+export const DeleteFriendModal: React.FC<DeleteFriendModalProps> = ({ name, children, companionCounter }) => {
+    return (
+        <div className={classes.modalContent}>
+            {name && (
+                <p className={classes.message}>
+                    Удалить из компанионов <br /> {name}?
+                </p>
+            )}
+
+            {companionCounter && <p className={classes.message}>Удалить {companionCounter} компанионов?</p>}
```

Comment:
Если companionCounter === 0, то выведет 0 :) Сам недавно попался в проекте.

```suggestion
            {Boolean(companionCounter) && <p className={classes.message}>Удалить {companionCounter} компанионов?</p>}
```

Reply:
![Снимок экрана_20250701_133504](https://github.com/user-attachments/assets/f57a6626-8780-4d01-a6c8-e8c39ebc0e10)
Ты прав :)